.github/workflows/wasm-size.yml

```diff
@@ -44,18 +47,21 @@ jobs:
     steps:
       - name: Checkout base branch
         uses: actions/checkout@v4
-        with:
-          ref: ${{ github.event.pull_request.base.sha }}
+        # with:
```

Comment:
This will be subject of removal in a follow-up PR. It's harder to make it backwards compatible. Basically,  the problem in here is that the refactoring of the nix derivations make it so that running this over main does no longer work. To test that this was actually correct, we are comparing the same size of the same commits. Once this is merged into main, we can revert this particular change, and do the right base..head comparison.